### DISCUSSION TITLE
Folge Modus / Follow mode

### DISCUSSION THREAD

**Discussion by ToToGSARider** [devemux86/cruiser#317] [Q&A] [answered]
Ist es für die Zukunft angedacht eine 3D Ansicht im Folge Modus zu haben. Hier ist ja aktuell nur 2D möglich. 

Is it planned for the future to have a 3D view in follow mode. Currently only 2D is possible here.

**Comment by devemux86** [comment, marked as answer]
@ToToGSARider 

Long press the compass and you can select GPS 2D or 3D mode.

Durch langes Drücken auf den Kompass können Sie den GPS 2D- oder 3D-Modus auswählen.

**Reply by ToToGSARider** [reply]
Dankeschön
